Earthfile

```diff
@@ -433,21 +441,32 @@ netboot:
 
 arm-image:
   ARG OSBUILDER_IMAGE
+  ARG COMPRESS_IMG=true
   FROM $OSBUILDER_IMAGE
   ARG MODEL=rpi64
   ARG IMAGE_NAME=${FLAVOR}.img
   WORKDIR /build
-  ENV STATE_SIZE="6200"
-  ENV RECOVERY_SIZE="4200"
   ENV SIZE="15200"
-  ENV DEFAULT_ACTIVE_SIZE="2000"
+  IF [[ "$FLAVOR" =~ ^ubuntu* ]]
+    ENV STATE_SIZE="6900"
+    ENV RECOVERY_SIZE="4600"
+    ENV DEFAULT_ACTIVE_SIZE="2300"
```

Comment:
Probably outside the scope of this PR, but it'd be nice if these `_SIZE` variables had the expected unit in their name.  It'd make it much more approachable by new folks.

Reply:
Created an issue since it also requires changes in osbuilder

https://github.com/kairos-io/kairos/issues/1397